core/shortcut_registry.ts

```diff
@@ -92,27 +88,31 @@ export class ShortcutRegistry {
   /**
    * Adds a mapping between a keycode and a keyboard shortcut.
    *
+   * If allowCollisions is used to mapped a single keycode to multiple
+   * shortcuts, they will be processed in last-registered,
```

Comment:
I don't understand what 'last-registered, first-activated order' means. Can you spell it out more?